src/SFML/System/Time.cpp

```diff
@@ -31,230 +31,8 @@
-bool operator >=(Time left, Time right)
-{
-    return left.asMicroseconds() >= right.asMicroseconds();
-}
-
-
-////////////////////////////////////////////////////////////
-Time operator -(Time right)
-{
-    return microseconds(-right.asMicroseconds());
-}
-
-
-////////////////////////////////////////////////////////////
-Time operator +(Time left, Time right)
-{
-    return microseconds(left.asMicroseconds() + right.asMicroseconds());
-}
-
-
-////////////////////////////////////////////////////////////
-Time& operator +=(Time& left, Time right)
-{
-    return left = left + right;
-}
-
-
-////////////////////////////////////////////////////////////
-Time operator -(Time left, Time right)
-{
-    return microseconds(left.asMicroseconds() - right.asMicroseconds());
-}
-
-
-////////////////////////////////////////////////////////////
-Time& operator -=(Time& left, Time right)
-{
-    return left = left - right;
-}
-
-
-////////////////////////////////////////////////////////////
-Time operator *(Time left, float right)
-{
-    return seconds(left.asSeconds() * right);
-}
-
-
-////////////////////////////////////////////////////////////
-Time operator *(Time left, Int64 right)
-{
-    return microseconds(left.asMicroseconds() * right);
-}
-
-
-////////////////////////////////////////////////////////////
-Time operator *(float left, Time right)
-{
-    return right * left;
-}
-
-
-////////////////////////////////////////////////////////////
-Time operator *(Int64 left, Time right)
-{
-    return right * left;
-}
-
-
-////////////////////////////////////////////////////////////
-Time& operator *=(Time& left, float right)
-{
-    return left = left * right;
-}
-
-
-////////////////////////////////////////////////////////////
-Time& operator *=(Time& left, Int64 right)
-{
-    return left = left * right;
-}
-
-
-////////////////////////////////////////////////////////////
-Time operator /(Time left, float right)
-{
-    return seconds(left.asSeconds() / right);
-}
-
-
-////////////////////////////////////////////////////////////
-Time operator /(Time left, Int64 right)
-{
-    return microseconds(left.asMicroseconds() / right);
-}
-
-
-////////////////////////////////////////////////////////////
-Time& operator /=(Time& left, float right)
-{
-    return left = left / right;
-}
-
-
-////////////////////////////////////////////////////////////
-Time& operator /=(Time& left, Int64 right)
-{
-    return left = left / right;
-}
-
-
-////////////////////////////////////////////////////////////
-float operator /(Time left, Time right)
-{
-    return left.asSeconds() / right.asSeconds();
-}
-
-
-////////////////////////////////////////////////////////////
-Time operator %(Time left, Time right)
-{
-    return microseconds(left.asMicroseconds() % right.asMicroseconds());
-}
-
-
-////////////////////////////////////////////////////////////
-Time& operator %=(Time& left, Time right)
-{
-    return left = left % right;
-}
+const Time Time::Zero;
```

Comment:
If we added this in the header, we could rename the whole file .cpp -> .inl, and wouldn't need to export any more symbols.

```cpp
constexpr Time Time::Zero(0);
```

Example:
https://onlinegdb.com/5uK6_OF8a

Reply:
This is the same problem as with `sf::Color`, see #1904 for discussion. I remember trying the workaround you suggested, but it failed CI at one point. I will retry in a separate PR for both `sf::Color` and `sf::Time`.

Reply:
Yes, I thought might be easier for you than having to do the administrative stuff twice. Wasn't aware of CI problems.

But your suggestion sounds good to me👍

Reply:
@Bromeon: please don't forget to approve otherwise I cannot merge :)

Reply:
@Bromeon: ping